### DISCUSSION TITLE
how to make Chinese entity linking training data?

### DISCUSSION THREAD

**Discussion by Zhou0418** [explosion/spaCy#7665] [Help: Coding & Implementations] [answered]
![image](https://user-images.githubusercontent.com/29883149/113662095-fc4e6d80-96d9-11eb-86f1-af77277502ab.png)
it's the data format for entity linking train. is that right?

- GitHub Discussions: https://github.com/explosion/spaCy/discussions
- Stack Overflow: http://stackoverflow.com/questions/tagged/spacy
-->

## Your Environment

- Operating System:
- Python Version Used:
- spaCy Version Used:
- Environment Information:

**Comment by svlandeg** [comment]
Hi! This question is well suited for the discussions forum so I'll move it there. The original issue will be closed, but will link through to the open discussion thread.

**Comment by svlandeg** [comment, marked as answer]
Yes, that looks correct to me, as long as chars 0-5 align with a token boundary - a named entity always spans one or more tokens (words).

Are your `sent_starts` correct though? They indicate that the first 3 tokens are sentences by themselves.

**Reply by Zhou0418** [reply]
it can be trained
but i still changed sent_starts, i just set the entitie's first char as 1 and the last one as -1, the others all set 0. the result is better than before

**Reply by svlandeg** [reply]
Yea, that's what I thought. The entity linker takes the context of the full sentence, so (artificially) short sentences would negatively impact performance. Good to hear it's working!
